libr/arch/p/arm/armass64.c

```diff
@@ -2016,21 +2016,41 @@ bool arm64ass(const char *str, ut64 addr, ut32 *op) {
 	} else if (!strncmp (str, "b ", 2)) {
 		*op = branch (&ops, addr, 0x14);
 #if 0
-02 bhs
-03 blo
-04 bmi
-05 bpl
-06 bvs
-07 bvc
-08 bhi
-09 bls
+02 bhs DONE
```

Comment:
just remove the whole comment :) it's a todo that doesnt needs to stay in the code. thanks!